### ISSUE TITLE
[BUG] webkit does not lunch since 1.33 using docker image

### ISSUE THREAD

**Issue by marc-mabe** [microsoft/playwright#22840]
### System info
- Playwright Version: [v1.33]
- Operating System: [amazon-ebs.ubuntu-focal-2004 (host) -> Ubuntu 22.04.2 (docker image)]
- Browser: [WebKit]
- Other info:

I'm using your docker image (see below) which works fine locally using Ubuntu 22.04 as host but it constantly fails since 1.33 on jenkins agent which is running on AWS t3.large using amazon-ebs.ubuntu-focal-2004.

### Source code

- [x] I provided exact source code that allows reproducing the issue locally.

**Dockerfile**
```
# docker/playwright/Dockerfile
FROM mcr.microsoft.com/playwright:v1.33.0

ENV HOME=/home/pwuser
RUN chmod -R 777 /home/pwuser

ENTRYPOINT ["npx", "playwright"]
CMD ["-h"]
```

**docker-compose**
```
# docker-compose.yml
version: "2.4"
services:
  playwright:
    user: 1000:1000
    ipc: host
    working_dir: /app
    environment:
      APP_BASEURL: "http://php-dev:80"
    build:
      context: ./
      dockerfile: docker/playwright/Dockerfile
    volumes:
      - ./:/app
```

**Config file**
```js
// playwright.config.ts
import type { PlaywrightTestConfig } from '@playwright/test';
import { devices } from '@playwright/test';

const config: PlaywrightTestConfig = {
  testDir: './playwright',
  timeout: 60 * 1000,
  expect: {
    timeout: 5000
  },
  fullyParallel: true,
  forbidOnly: false,
  retries: 3,
  workers: require('os').cpus().length,
  reporter:[ ['html', { open: 'never', outputFolder: './var/playwright-report' }] ],
  use: {
    actionTimeout: 0,
    baseURL: process.env.APP_BASEURL ?? 'http://php-dev:80',
    extraHTTPHeaders: {
      'X-Debug': '0', // Disable symfony debug toolbar (profiler)
    },
    trace: 'on',
    video: 'on-first-retry',
    launchOptions: {
      slowMo: 300
    }
  },
  outputDir: './var/playwright-artifacts',
};

export default config;
```

**Test file (self-contained)**
Does not matter as the browser does not start

**Steps**
```
$ docker-compose run --rm --entrypoint '' playwright /bin/bash -c 'DEBUG=pw:browser npx playwright test --browser "webkit"'
Creating XXX ... done
Running 5 tests using 2 workers
[webkit] › xxx1.spec.ts:13:9 › test1
  pw:browser <launching> /ms-playwright/webkit-1837/pw_run.sh --inspector-pipe --headless --no-startup-window +0ms
  pw:browser <launched> pid=871 +16ms
[webkit] › xxx2.spec.ts:5:5 › test2
  pw:browser <launching> /ms-playwright/webkit-1837/pw_run.sh --inspector-pipe --headless --no-startup-window +0ms
  pw:browser <launched> pid=894
```
At this poing it just hangs until the process gets killed manually.

**Expected**
Browser starts and tests start executing

**Actual**
Webkit hangs on lunch

**Comment by mxschmitt** [comment]
Most likely a duplicate of https://github.com/microsoft/playwright/issues/15764#issuecomment-1462787720.

TL;DR: There is a bug that WebKit will hang on startup when you use Ubuntu22 Docker container on a Ubuntu20 host. You can either apply the workaround from the linked comment above or try to migrate to Ubuntu22 Docker host.
